docfx_yaml/extension.py

```diff
@@ -322,6 +322,10 @@ def build_finished(app, exception):
                 # Skip objects without a module
                 continue
 
+            if filename.lower().find('.tests') != -1:
+                # Skip *.tests*.yml files
+                continue
+
```

Comment:
I worry this might catch legitimate files that are named `tests`. This is fine for now, but it should be broken out into a configurable way to exclude files, if it's to be used by other projects.